### PR TITLE
DC, Storage and OS Migration

### PR DESCRIPTION
**Pull request by sanjay-boddu**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -21,10 +21,10 @@ Markdown==2.6.11
 MarkupSafe==2.1.0
```
Comment (bilalebi) [inline requirements-dev.txt:21]: We can remove libraries duplication by importing `requirements.txt` as we did in [Thoas](https://github.com/Ensembl/ensembl-thoas/blob/main/requirements-dev.txt).
Where we put common packages in  `requirements.txt` and the rest (related to `dev`) here in `requirements-dev.txt`

File: tark/tark/settings/env.py [hunk 2]
```diff
@@ -19,5 +19,5 @@
 
 ENV = os.getenv('DJANGO_ENVIRONMENT', 'development')
```
Comment (bilalebi) [inline tark/tark/settings/env.py:20]: For the sake of consistency, I think this should be either `dev` or `development` based on what we decide to use here (in [tark-configs repo](https://github.com/Ensembl/tark-configs)):
```
# Replace the following with 'dev' or 'staging' or 'primary'
Define PROJECT_ENV REPLACE_ME
```

**Review by bilalebi — CHANGES_REQUESTED** [review]

File: requirements-dev.txt (hunk 1 above)
Comment (sanjay-boddu) [reply requirements-dev.txt:21]: Sounds good. Will implement that.

File: tark/tark/settings/env.py (hunk 2 above)
Comment (sanjay-boddu) [reply tark/tark/settings/env.py:20]: Updated in config repo

**Review by ahc87 — COMMENTED** [review]
The `local.py` file that you're removing references some logging stuff, and also `INTERNAL_IPS`: https://github.com/Ensembl/tark/blob/main/tark/tark/settings/local.py#L33-L51.  Is `INTERNAL_IPS` needed, and will we still be able to see logging after this change?

Also, the README references the `secrets_template.py` and the logging, so you will need to update the README to match the changes here.

File: requirements-dev.txt [hunk 3]
```diff
@@ -21,10 +21,10 @@ Markdown==2.6.11
...
 mysql-connector-python==8.0.28
 mysqlclient==2.1.0
-numpy==1.21.0
+numpy==1.19.5
```
[the remark this answers is not in the document]
Comment (sanjay-boddu) [reply requirements-dev.txt:24]: Yes, compatibility with python3.6

File: README.md [hunk 4]
```diff
@@ -40,44 +39,20 @@ pip install -r requirements.txt
...
-            'HOST': 'localhost',
-            'PORT': '3306',
-
-```
-
-
-Update the log file location in `local.py` (point this to somewhere writable location)
-```
-LOG_FILE = os.path.join(BASE_DIR, '../../logs/tark.log')
+cp tark/tark/settings/secrets_template.py tark/tark/settings/secrest.py
```
Comment (ahc87) [inline README.md:42]: `secrest.py` -> `secrets.py`

**Comment by sanjay-boddu** [comment]
> The `local.py` file that you're removing references some logging stuff, and also `INTERNAL_IPS`: https://github.com/Ensembl/tark/blob/main/tark/tark/settings/local.py#L33-L51. Is `INTERNAL_IPS` needed, and will we still be able to see logging after this change?
> 
> Also, the README references the `secrets_template.py` and the logging, so you will need to update the README to match the changes here.


Thanks, updated README file.

File: README.md (hunk 4 above)
Comment (sanjay-boddu) [reply README.md]: Done :)

File: requirements-dev.txt (hunk 1 above)
Comment (sanjay-boddu) [reply requirements-dev.txt:21]: @bilalebi , creating requirements-dev.txt may be out of the scope of the DC migration. I have created a ticket for this to work on after migration

Comment (bilalebi) [reply requirements-dev.txt:21]: @sanjay-boddu, I agree 👍

**Review by bilalebi — APPROVED** [review]

File: tark/tark/settings/env.py [hunk 5]
```diff
@@ -19,5 +19,5 @@
...
 DEV_ENV = ENV == 'development'
-TEST_ENV = ENV == 'staging'
-PROD_ENV = ENV == 'production'
+STAGING_ENV = ENV == 'staging'
+PROD_ENV = ENV == 'primary'
```
Comment (ahc87) [inline tark/tark/settings/env.py:23]: Why change 'production' to 'primary'?  Could we update "ENV setup" on this page to match? https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?pageId=111295328

Comment (sanjay-boddu) [reply tark/tark/settings/env.py]: That confluence page will go. We will have a new setup/deployment SOP(WIP)

File: tark/tark/settings/local.py [hunk 6]
```diff
@@ -1,82 +0,0 @@
...
-            'level': 'DEBUG',
-            'propagate': True,
-        },
-    },
-}
-
-if DEBUG:
-    INTERNAL_IPS = ('127.0.0.1', 'localhost')
-
-if DEBUG_TOOLBAR:
```
Comment (sanjay-boddu) [inline tark/tark/settings/local.py:56]: Add to base.py

**Pull request merged**